tests/test_recipient_csv.py

```diff
@@ -708,21 +708,10 @@ def test_bad_or_missing_data(
 @pytest.mark.parametrize(
     "file_contents,rows_with_bad_recipients",
     [
-        (
-            """
-            phone number
-            800000000000
-            1234
-            +447900123
-        """,
-            {0, 1, 2},
-        ),
         (
             """
             phone number, country
-            1-202-555-0104, USA
-            +12025550104, USA
```

Comment:
why reduction in lines and test cases?

Looks like we are only testing happy path now?

Reply:
These pass, it was just for ease of testing one edge case. They'll be added back in